packages/kbn-lens-embeddable-utils/attribute_builder/visualization_types/layers/xy_data_layer.ts

```diff
@@ -48,6 +48,9 @@ export interface XYLayerOptions {
   // Add more types as support for them is implemented
   buckets?: DateHistogramBucketedColumn;
   seriesType?: SeriesType;
+  params?: {
+    interval: string;
+  };
```

Comment:
`interval` is already in the `params` type, declared in the `DateHistogramColumnParams` type. We don't need this one here.

You can set its value like this:

```ts
  const xYDataLayerOptions: XYLayerOptions = {
      buckets: { type: 'date_histogram', params: { interval: `${timeSize}${timeUnit}` } },
      seriesType: 'bar',
    };
```